data/tlg0341/tlg002/tlg0341.tlg002.perseus-grc1.xml

```diff
@@ -1,1606 +1,1604 @@
+                <l n="1396">ὀθνεῖα γατομοῦντος Αἴθωνος πτερά.</l>
+                <l n="1397">ὁ Φρὺξ δʼ, ἀδελφὸν αἷμα τιμωρούμενος,</l>
+                <l n="1398">πάλιν τιθηνὸν ἀντιπορθήσει χθόνα</l>
+                <l n="1399">τοῦ νεκροτάγου, τὰς ἀθωπεύτους δίκας</l>
+                <l n="1400">φθιτοῖσι ῥητρεύοντος ἀστεργεῖ τρόπῳ. </l>
+                <l n="1401">ὃς δή ποτʼ ἀμφώδοντος ἐξ ἄκρων λοβῶν</l>
+                <l n="1402">φθέρσας κύφελλα καλλυνεῖ παρωτίδας,</l>
+                <l n="1403">δαπταῖς τιτύσκων αἱμοπώταισιν φόβον.</l>
+                <l n="1404">τῷ πᾶσα Φλεγρὰς αἶα δουλωθήσεται</l>
+                <l n="1405">Θραμβουσία τε δειρὰς ἥ τʼ ἐπάκτιος </l>
+                <l n="1406">στόρθυγξ Τίτωνος αἵ τε Σιθόνων πλάκες</l>
+                <l n="1407">Παλληνία τʼ ἄρουρα, τὴν ὁ βούκερως</l>
+                <l n="1408">Βρύχων λιπαίνει, γηγενῶν ὑπηρέτης.</l>
+                <l n="1409">πολλῶν δʼ ἐναλλὰξ πημάτων ἀπάρξεται</l>
+                <l n="1410">Κανδαῖος ἢ Μάμερτος, ἢ τί χρὴ καλεῖν </l>
+                <l n="1411">τὸν αἱμοφύρτοις ἑστιώμενον μάχαις;</l>
+                <pb n="612" />
+                <l n="1412">οὐ μὰν ὑπείξει γʼ ἡ ʼπιμηθέως τοκάς,</l>
+                <l n="1413">ἀλλʼ ἀντὶ πάντων Περσέως ἕνα σπορᾶς</l>
+                <l n="1414">στελεῖ γίγαντα, τῷ θάλασσα μὲν βατὴ</l>
+                <l n="1415">πεζῷ ποτʼ ἔσται, γῆ δὲ ναυσθλωθήσεται </l>
+                <l n="1416">ῥήσσοντι πηδοῖς χέρσον. οἱ δὲ Λαφρίας</l>
+                <l n="1417">οἶκοι Μαμέρσας, ᾐθαλωμένοι φλογὶ</l>
+                <l n="1418">σὺν καλίνοισι τειχέων προβλήμασι</l>
+                <l n="1419">τὸν χρησμολέσχην αἰτιάσονται βλάβης,</l>
+                <l n="1420">ψαίνυνθα θεσπίζοντα Πλούτωνος λάτριν. </l>
+                <l n="1421">στρατῷ δʼ ἀμίκτῳ πᾶσα μὲν βρωθήσεται,</l>
+                <l n="1422">φλοιῶτιν ἐκδύνουσα δίπλακα σκέπην,</l>
+                <l n="1423">καρποτρόφος δρῦς ἀγριάς τʼ ὀρειθαλής.</l>
+                <l n="1424">ἅπας δʼ ἀναύρων νασμὸς αὐανθήσεται,</l>
+                <l n="1425">χανδὸν κελαινὴν δίψαν αἰονωμένων. </l>
+                <l n="1426">κύφελλα δʼ ἰῶν τηλόθεν ῥοιζουμένων</l>
+                <l n="1427">ὑπὲρ κάρα σιήσουσι, Κίμμερός θʼ ὅπως,</l>
+                <l n="1428">σκιὰ καλύψει πέρραν, ἀμβλύνων σέλας.</l>
+                <l n="1429">Λοκρὸν δʼ ὁποῖα παῦρον ἀνθήσας ῥόδον,</l>
+                <l n="1430">καὶ πάντα φλέξας, ὥστε κάγκανον στάχυν, </l>
+                <l n="1431">αὖθις παλιμπλώτοιο γεύσεται φυγῆς,</l>
+                <l n="1432">μόσσυνα φηγότευκτον, ὡς λυκοψίαν</l>
+                <l n="1433">κόρη κνεφαίαν, ἄγχι παμφαλώμενος,</l>
+                <l n="1434">χαλκηλάτῳ κνώδοντι δειματουμένη.</l>
+                <l n="1435">πολλοὶ δʼ ἀγῶνες καὶ φόνοι μεταίχμιοι </l>
+                <l n="1436">λύσουσιν ἀνδρῶν οἱ μὲν ἐν γαίᾳ <note n="1" place="unspecified">One
+                    expects &quot;on the sea,&quot; but no satisfactory
```

Comment:
No entities should remain. `&quot;` should be double curly quote (if that is print rendering) or `<q>`